src/css/style.css

```diff
@@ -48,67 +79,212 @@ p{
 }
 
 /* "header start" */
-#header{}
+#header {}
+
 /* "header end" */
 
+
 /* "main-block start" */
-#main-block{}
+#main-block {
+    position: relative;
+}
+
+#main-block .background-container {
+    position: relative;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-image: url('../Photo/Bgmobile.png');
+    height: 362px;
+}
+
+#main-block .h1 {
+    width: 287px;
+    margin: 101px 0 27px 0;
+}
+
+#main-block .main {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+#main-block p {
+    font-family: 'Ubuntu';
```

Comment:
What's "Ubuntu"? We are using Bold/Regular/Medium as well.
Also please read code conventions, we are using " instead of '